website/screens/components/footer/specs/FooterSpecsPage.tsx

```diff
@@ -0,0 +1,456 @@
+import DocFooter from "@/common/DocFooter";
+import Figure from "@/common/Figure";
+import HeadingLink from "@/common/HeadingLink";
+import Image from "@/common/Image";
+import {
+  DxcList,
+  DxcStack,
+  DxcTable,
+  DxcText,
+} from "@dxc-technology/halstack-react";
+import Code from "../../../common/Code";
+import footerAnatomy from "./images/footer_anatomy.png";
+import footerSpecs from "./images/footer_specs.png";
+
+const FooterSpecsPage = () => {
+  return (
+    <DxcStack gutter="xxxlarge">
+      <DxcStack gutter="large">
+        <HeadingLink level={2}>Specifications</HeadingLink>
+        <Figure caption="Footer component design specs">
+          <Image src={footerSpecs} alt="Footer component design specs" />
+        </Figure>
+      </DxcStack>
+      <DxcStack gutter="large">
+        <HeadingLink level={3}>Anatomy</HeadingLink>
+        <Figure caption="Footer anatomy">
```

Comment:
The original documentation doesn't have any `caption` for this image. I suggest removing the `<Figure>` and having only the `<Image>`.

Reply:
Most components have a caption for anatomy image. I think it is better to keep it.